docs/xml/elements/contest_base.rst

```diff
@@ -1,37 +1,103 @@
++-------------------------+---------------------------------+-----------+----------+--------------------------+----------------------+
+| ExternalIdentifiers     |:doc:`ExternalIdentifiers        | Optional  | Single   |Another identifier for a  |If the element is     |
+|                         |<external_identifiers>`          |           |          |contest that links to     |invalid or not        |
+|                         |                                 |           |          |another source of         |present, the          |
+|                         |                                 |           |          |information.              |implementation should |
+|                         |                                 |           |          |                          |ignore it.            |
+|                         |                                 |           |          |                          |                      |
+|                         |                                 |           |          |                          |                      |
++-------------------------+---------------------------------+-----------+----------+--------------------------+----------------------+
+| HasRotation             | xs:boolean                      | Optional  | Single   |Indicates whether the     |If the field is       |
+|                         |                                 |           |          |selections in the contest |invalid or not        |
+|                         |                                 |           |          |are rotated.              |present, the          |
+|                         |                                 |           |          |                          |implementation should |
+|                         |                                 |           |          |                          |ignore it.            |
++-------------------------+---------------------------------+-----------+----------+--------------------------+----------------------+
+| Name                    | xs:string                       | Optional  | Single   |Name of the contest, not  |If the field is       |
+|                         |                                 |           |          |necessarily how it appears|invalid or not        |
+|                         |                                 |           |          |on the ballot (NB:        |present, the          |
+|                         |                                 |           |          |BallotTitle should be used|implementation should |
+|                         |                                 |           |          |for this purpose).        |ignore it.            |
++-------------------------+---------------------------------+-----------+----------+--------------------------+----------------------+
+| SequenceOrder           | xs:integer                      | Optional  | Single   |Order in which the        |If the field is       |
```

Comment:
Actually I believe this is the order in which the contest shows up on the ballot. This can be used if no `BallotStyle` elements are linked to a given `Precinct`, but I'll double-check with @johnpwack about its use in 1622.2, since the Michigan sample feed uses both `SequenceOrder` and `BallotStyle` elements.